collector/lib/CollectorConfig.h

```diff
@@ -131,6 +132,7 @@ end
   bool enable_afterglow_ = true;
   bool enable_core_dump_ = false;
   bool enable_processes_listening_on_ports_;
+  bool are_udp_listening_endpoints_collected_ = false;
```

Comment:
I find it simpler when the env-var is the only place where we set the default value.

Reply:
I am not 100 % sure what you mean. If the environment variable is not set, we need a default value in the code.